src/components/profile/MyProfile.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+import ProfileMissions from '../missions/ProfileMissions';
+import style from './Profile.module.css';
+
+const Profile = () => {
+// eslint-disable-next-line max-len
```

Comment:
- [ ] Unnecessary use of eslint-disable. kindly consider indenting this statement to improve readability